examples/websocket-relay/relay-app/src/stores-original.ts.bak

```diff
@@ -0,0 +1,271 @@
+import { derived, writable } from "svelte/store";
```

Comment:
Do we still need this file @bgins?

Reply:
Nope! We can get rid of that entire file, was just using to keep some of the original code around.

Reply:
💥